httpony/server.py

```diff
@@ -24,6 +27,30 @@ def main():
     app = make_app()
     run_simple(hostname, port, app)
 
+def parse(argv):
+    """Parse the user arguments."""
+    parser = build_parser()
+
+    # argparse expects the executable to be removed from argv.
+    args = parser.parse_args(argv[1:])
+
+    return args
+
+def build_parser():
+    """Build the parser that will have all available commands and options."""
+    description = 'HTTPony (pronounced aych-tee-tee-pony) is a simple HTTP ' \
+                  'server that pretty prints HTTP requests to a terminal. It' \
+                  ' is a useful aide for developing clients that send HTTP ' \
+                  'requests. HTTPony acts as a sink for a client so that a ' \
+                  'developer can understand what the client is sending.'
+    parser = argparse.ArgumentParser(description=description)
+    parser.add_argument(
+        '-l', '--listen', help='set ip/hostname',
+        default='localhost')
+    parser.add_argument(
+        '-p','--port', help='set port', default='8000', type=int)
+
+    return parser
 
 if __name__ == '__main__':
-    main()
+    main()
```

Comment:
Well this is interesting.  During the meetup on saturday I was under the impression you didnt want new lines? I removed the line below main() and thats the change it made.